client/ayon_core/hosts/maya/plugins/publish/collect_user_defined_attributes.py

```diff
@@ -14,7 +14,10 @@ class CollectUserDefinedAttributes(pyblish.api.InstancePlugin):
     def process(self, instance):
 
         # Collect user defined attributes.
-        if not instance.data.get("includeUserDefinedAttributes", False):
+        if not instance.data.get(
+            "includeUserDefinedAttributes",
+            instance.data["creator_attributes"]["includeUserDefinedAttributes"]
+        ):
```

Comment:
This basically enforces the `["creator_attributes"]["includeUserDefinedAttributes"]` key to exist. I don't see any other usages of `instance.data["includeUserDefinedAttributes"]` in the code base that still require it to exist in `instance.data` so I would just refactor this to using the creator attributes value instead.

```suggestion
        if not instance.data["creator_attributes"].get(
            "includeUserDefinedAttributes"
        ):
```